python/paddle/fluid/layers/control_flow.py

```diff
@@ -352,7 +350,8 @@ def __init__(self, name=None):
 
     def step(self):
         """
-        The block for user to define operators in RNN.
+        Define operators in each step. step is used in with block, OP in with block
```

Comment:
:code:`with` or "with", otherwise people may be confused about the "with" block here

Reply:
sure , good idea